perun-cli/resumePropagation

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/perl
```

Comment:
I would prefer to have same naming scheme for both tools. Please rename this file to `resumeTasksPropagation`.